package.json

```diff
@@ -57,7 +57,7 @@
     "redux": "^3.6.0",
     "redux-auth-wrapper": "^1.0.0",
     "redux-thunk": "^2.2.0",
-    "uport-connect": "^0.6.1"
+    "uport-connect": "^0.7.3"
```

Comment:
perhaps use "latest" to avoid the need for future updates? ,

either way thank you for noticing this.

Reply:
Ship it!  I can speak to pro's/con's too... I thought it may ease the maintenance of this repo but I'm not opinionated on it.   I don't think we plan on publishing unstable releases.

We'll be pinging this repo again soon once our new library updates for DID's are released, which will contain some API changes needing another version bump.